samples/sensor/dht_polling/sample.yaml

```diff
@@ -15,6 +15,7 @@ tests:
       - nucleo_f401re
     harness: console
     harness_config:
+      fixture: fixture_i2c_hs300x
```

Comment:
The sample can run against any dht sensor though like the bme280, can we rename this fixture to match that expectation of dht_fixture or something along these lines?